tst/EnergyPlus/unit/ZoneTempPredictorCorrector.unit.cc

```diff
@@ -1059,6 +1058,16 @@ TEST_F(EnergyPlusFixture, ZoneTempPredictorCorrector_CalcZoneSums_SurfConvection
     EXPECT_EQ(10.0, SumHA);
     EXPECT_EQ(300.0, SumHATsurf);
     EXPECT_EQ(50.0, SumHATref);
+
+    Node(1).MassFlowRate = 0.1;
+    Node(2).MassFlowRate = 0.2;
+    CalcZoneSums(state.dataZonePlenum, ZoneNum, SumIntGain, SumHA, SumHATsurf, SumHATref, SumMCp, SumMCpT, SumSysMCp, SumSysMCpT);
+    EXPECT_NEAR(302.00968500, SumSysMCp, 0.0001);
+    EXPECT_NEAR(6040.1937, SumSysMCpT,0.0001);
+
+    CalcZoneSums(state.dataZonePlenum, ZoneNum, SumIntGain, SumHA, SumHATsurf, SumHATref, SumMCp, SumMCpT, SumSysMCp, SumSysMCpT, false);
+    EXPECT_EQ(0.0, SumSysMCp);
+    EXPECT_EQ(0.0, SumSysMCpT);
```

Comment:
Seems like a sufficient unit test.  In the first case we are doing the corrector and we calculate the system mCp and mCpT terms.  In the second case we aren't doing the corrector and so the values come back zero for the system.